model/api/pom.xml

```diff
@@ -53,6 +53,12 @@
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-annotations</artifactId>
       </dependency>
+        <dependency>
```

Comment:
Why we need this in the model?

Reply:
It has utility classes for validating the variant.